src/dataflow/src/sink/kafka.rs

```diff
@@ -422,10 +434,18 @@ where
             }
         });
 
+        // Figure out the durablity frontier for all sources we depent on
+        let mut durability_frontier = Antichain::new();
+
+        for history in source_timestamp_histories.iter() {
+            durability_frontier.extend(history.durability_frontier().elements().iter().cloned());
```

Comment:
i think what i'm doing here is a weird handrolled `meet_assign`

Reply:
Is there a reason not to do a real `meet_assign`?